cob_bringup/robots/cob4-5.xml

```diff
@@ -11,6 +11,7 @@
 	<arg name="cob4-5-h1" default="localhost"/>
 	<arg name="env-script" default="$(find cob_bringup)/env.sh"/>
 	<arg name="sim" default="false"/>
+	<arg name="cartesian_control" default="false"/>
```

Comment:
remove this as you cannot call a xml directly thus you cannot pass a launch argument..